### ISSUE TITLE
Fix: Accessibility, Responsiveness & UI Improvements

### ISSUE THREAD

**Issue by FNICKE** [Varshitha713/first-contrib-placement#200]
This PR fixes the following issues:
1. Added missing alt text to images
2. Removed unused/duplicate CSS
3. Added favicon
4. Added responsive meta tag
5. Standardized navigation bar
6. Added unique page titles
7. Added consistent footer
8. Fixed broken links
9. Added hover effects
10. Improved heading structure

**Comment by Varshitha713** [comment]
@FNICKE the favicon already exists if you have better suggestions please join the discord and share your design and you can work on the other things as mentioned in your issue!!
assigning it to you!!

**Comment by niveditasargar** [comment]
Hii , @Varshitha713 @FNICKE
 I would like to work on this issue and implement the mentioned fixes. Could you please assign it to me?
